src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets

```diff
@@ -180,11 +181,14 @@ Copyright (c) .NET Foundation. All rights reserved.
       <_WasmEnableThreads Condition="'$(_WasmEnableThreads)' == ''">false</_WasmEnableThreads>
 
       <_WasmEnableWebcil>$(WasmEnableWebcil)</_WasmEnableWebcil>
-      <_WasmEnableWebcil Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp' or '$(_TargetingNET80OrLater)' != 'true'">false</_WasmEnableWebcil>
+      <_WasmEnableWebcil Condition="'$(_TargetingNET80OrLater)' != 'true'">false</_WasmEnableWebcil>
       <_WasmEnableWebcil Condition="'$(_WasmEnableWebcil)' == ''">true</_WasmEnableWebcil>
       <_BlazorWebAssemblyStartupMemoryCache>$(BlazorWebAssemblyStartupMemoryCache)</_BlazorWebAssemblyStartupMemoryCache>
       <_BlazorWebAssemblyJiterpreter>$(BlazorWebAssemblyJiterpreter)</_BlazorWebAssemblyJiterpreter>
       <_BlazorWebAssemblyRuntimeOptions>$(BlazorWebAssemblyRuntimeOptions)</_BlazorWebAssemblyRuntimeOptions>
+      <_WasmDebugLevel>$(WasmDebugLevel)</_WasmDebugLevel>
+      <_WasmDebugLevel Condition="'$(_TargetingNET80OrLater)' == 'true' and '$(_WasmDebugLevel)' == ''">0</_WasmDebugLevel>
```

Comment:
This is because we don't want to "fix" this for Net7 right ?

Reply:
In .NET 7, blazor is consuming boot json and it will ignore that value... Maybe we can skip the condition. Most of the place don't care about values that are ignored in previous versions.